lib/addons/velero/index.ts

```diff
@@ -0,0 +1,227 @@
+import { ClusterAddOn, ClusterInfo } from "../../spi";
+import { Constants } from "..";
+import * as s3 from "@aws-cdk/aws-s3";
+import * as iam from "@aws-cdk/aws-iam";
+import { GatewayVpcEndpointAwsService } from "@aws-cdk/aws-ec2";
+import deepmerge = require('deepmerge');
+
+/**
+ * Configuration options for the add-on.
+ */
+export interface VeleroAddOnProps {
+    /**
+     * Velero for the Velero Helm Chart.
+     * @default 2.23.6
+     */
+    version?: string;
+   
+    /**
+     * Namespace for the add-on. If the namespace does not exist, specify create to false
+     * @default {namespace: "velero", create: true}
+     */
+    namespace?:  {
```

Comment:
sorry i think it is a bit confusing.  Changed to below format:
```json
    namespace?:  {
        name: string,   // default value is "velero", if it is specified then it will be the namespace of where velero is deployed to.
        create: boolean // default is true, if it is false, no new namespace will be created
    };
```